assets/blonkchain.js

```diff
@@ -0,0 +1,45 @@
+import jQuery from './assets/assets/assets/jquery.min.js';
```

Comment:
This isn't Hyperscript

Reply:
I feel like you are =judging my code,and it makes me feel bad. You are going against code of conduct. Ima cancel your doxx.

Reply:
@jorqensen I believe the newest version of hyperscript has an addon you can use within your CI/CD that can recombobulate the jquery into proper hyperscript syntax. This could help expand the availability of who can PR amazing additions like @sonja-turo and his `blonkChain`, while allowing them the leniency of not needing to adopt the *easier* syntax we all know and love, **hyperscript**.